benchmarks/src/main/scala/akka/kafka/benchmarks/ReactiveKafkaConsumerBenchmarks.scala

```diff
@@ -166,7 +166,7 @@ object ReactiveKafkaConsumerBenchmarks extends LazyLogging with InflightMetrics
     val control = fixture.source
       .mapAsync(1) { m =>
         meter.mark()
-        m.committableOffset.commitInternal().map(_ => m)(ExecutionContexts.sameThreadExecutionContext)
+        m.committableOffset.commitInternal(flush = false).map(_ => m)(ExecutionContexts.sameThreadExecutionContext)
```

Comment:
Revert for new default param.  There are a bunch of these callsites that can be reverted.

Reply:
fixed, I walked through other occurrences as well